src/openforms/registrations/contrib/objects_api/checks.py

```diff
@@ -4,13 +4,23 @@
 
 from openforms.plugins.exceptions import InvalidPluginConfiguration
 
-from .client import NoServiceConfigured, get_documents_client, get_objects_client
+from .client import (
+    NoServiceConfigured,
+    get_documents_client,
+    get_objects_client,
+    get_objecttypes_client,
+)
 from .models import ObjectsAPIConfig
 
 
 def check_objects_service():
     with get_objects_client() as client:
-        client.get("objects")
+        client.get("objects", params={"pageSize": 1})
```

Comment:
What about doing a head request instead?

Reply:
`HEAD` is not listed in the API spec: https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/maykinmedia/objects-api/2.2.0/src/objects/api/v2/openapi.yaml#tag/objects

Reply:
Let's keep it this way then